Javascript/achievements.js

```diff
@@ -329,313 +327,532 @@ var areward182 = "Unlock Tier 8 Ant autobuy, and autobuy Scientia and Phylacteri
+    }
+    if (prestigePointGain.greaterThanOrEqualTo("1e1000")) {
+      achievementaward(39)
 
-        }
-        if (prestigePointGain.greaterThanOrEqualTo("1e10000")) {
-            achievementaward(40)
+    }
+    if (prestigePointGain.greaterThanOrEqualTo("1e10000")) {
+      achievementaward(40)
 
-        }
-        if (prestigePointGain.greaterThanOrEqualTo("1e77777")) {
-            achievementaward(41)
+    }
+    if (prestigePointGain.greaterThanOrEqualTo("1e77777")) {
+      achievementaward(41)
 
-        }
-        if (prestigePointGain.greaterThanOrEqualTo("1e250000")) {
-            achievementaward(42)
+    }
+    if (prestigePointGain.greaterThanOrEqualTo("1e250000")) {
+      achievementaward(42)
 
-        }
-    }
-    if (i == 2) {
-        if (player.transcendnoaccelerator == true) {
-            achievementaward(61)
-        }
-        if (player.transcendnomultiplier == true) {
-            achievementaward(58)
-        }
-        if (player.transcendnocoinupgrades == true) {
-            achievementaward(65)
-        }
-        if (player.transcendnocoinorprestigeupgrades == true) {
-            achievementaward(66)
-        }
-        if (player.transcendnoaccelerator == true) {
-            achievementaward(61)
-        }
-        if (player.transcendnomultiplier == true) {
-            achievementaward(58)
-        }
-        if (transcendPointGain.greaterThanOrEqualTo(1)) {
-            achievementaward(43)
-        }
-        if (transcendPointGain.greaterThanOrEqualTo(1e6)) {
-            achievementaward(44)
-        }
-        if (transcendPointGain.greaterThanOrEqualTo(1e50)) {
-            achievementaward(45)
-        }
-        if (transcendPointGain.greaterThanOrEqualTo(1e308)) {
-            achievementaward(46)
-        }
-        if (transcendPointGain.greaterThanOrEqualTo("1e1500")) {
-            achievementaward(47)
-        }
-        if (transcendPointGain.greaterThanOrEqualTo("1e25000")) {
-            achievementaward(48)
-        }
-        if (transcendPointGain.greaterThanOrEqualTo("1e100000")) {
-            achievementaward(49)
-        }
-    }
-    if (i == 3) {
-        if (player.reincarnatenoaccelerator == true) {
-            achievementaward(62)
-        }
-        if (player.reincarnatenomultiplier == true) {
-            achievementaward(59)
-        }
-        if (player.reincarnatenocoinupgrades == true) {
-            achievementaward(67)
-        }
-        if (player.reincarnatenocoinorprestigeupgrades == true) {
-            achievementaward(68)
-        }
-        if (player.reincarnatenocoinprestigeortranscendupgrades == true) {
-            achievementaward(69)
-        }
-        if (player.reincarnatenocoinprestigetranscendorgeneratorupgrades == true) {
-            achievementaward(70)
-        }
-        if (reincarnationPointGain.greaterThanOrEqualTo(1)) {
-            achievementaward(50)
+    }
+  }
+  if (i == 2) {
+    if (player.transcendnoaccelerator == true) {

```

Comment:
These can be triple eq (===); it will not cause any difference.

Reply:
Done